src/YOLO/utils.js

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2018 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+/* eslint max-len: ["error", { "code": 180 }] */
+
+function map(num, inmin, inmax) {
+  return ((num - inmin) * 360) / (inmax - inmin);
+}
+function hsl(num, min, max) {
+  return `hsla(${map(num, min, max)}, 100%, 50%,1)`;
+}
+function drawObject(item, ctx, color) {
+  const txt = `${item.label} : ${(item.score * 100).toFixed(2)}%`;
+  ctx.strokeStyle = color;
+  ctx.beginPath();
+  ctx.rect(item.x, item.y, item.w, item.h);
+  ctx.stroke();
+  ctx.fillStyle = color;
+  if (item.y - 25 >= 0) {
```

Comment:
you can replace all of this if-else with a ternary.
Also it's better to have < than >= or > than <=.

```JS
    const yLessThanZero = item.y - 25 < 0;

    ctx.fillRect(item.x - 1.5, item.y - 1.5, 90, yLessThanZero? 20 : -15);
    ctx.fillStyle = '#000000';
    ctx.fillText(txt, item.x, yLessThanZero ? item.y + 12 : item.y - 5 );
```